### PR TITLE
Bug fixes for the nios member object

### PR DESCRIPTION
**Pull request by krisvasudevan**
##### SUMMARY

Bug fix: Error while trying to create HA member and adding management network details using NIOS member module

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

nios_member

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/module_utils/net_tools/nios/api.py
Comment (justjais) [inline lib/ansible/module_utils/net_tools/nios/api.py]: @krisvasudevan I think you are trying to check if the `member_spec[key] is not None`, if yes plz modify the check as:
```suggestion
        if key in member_elements and member_spec[key] is not None:
```

**Review by justjais — COMMENTED** [review]
@krisvasudevan please close the following PR: https://github.com/ansible/ansible/pull/56043, as it's the duplicate one with shippable failure.

File: lib/ansible/module_utils/net_tools/nios/api.py
Comment (krisvasudevan) [reply lib/ansible/module_utils/net_tools/nios/api.py]: @justjais 
I've committed your changes. Thanks for taking the time out to review this.

Comment (justjais) [reply lib/ansible/module_utils/net_tools/nios/api.py]: @krisvasudevan changes lgtm, once CI passes I'll merge the PR, also as suggested in mail plz create a backport PR for the changes to take effect in `2.8.1`

**Review by justjais — APPROVED** [review]
+1

**Pull request merged**
